conn_test.go

```diff
@@ -166,17 +193,12 @@ var _ = Describe("Connection", func() {
 	})
 
 	It("filters addresses", func() {
-		filters := filter.NewFilters()
-		ipNet := net.IPNet{
-			IP:   net.IPv4(127, 0, 0, 1),
-			Mask: net.IPv4Mask(255, 255, 255, 255),
-		}
-		filters.AddFilter(ipNet, filter.ActionDeny)
 		testMA, err := ma.NewMultiaddr("/ip4/127.0.0.1/udp/1234/quic")
 		Expect(err).ToNot(HaveOccurred())
-		Expect(filters.AddrBlocked(testMA)).To(BeTrue())
+		cg := &localhostMockGater{}
+		Expect(cg.InterceptAccept(&connAddrs{rmAddr: testMA})).To(BeFalse())
```

Comment:
This is testing the gater, rather than the code we've introduced.

Reply:
Well, it seemed to be the case before too. I guess a sanity check.

Reply:
@raulk Yes, hence left it there.